api/src/main/java/keywhiz/api/model/SecretContent.java

```diff
@@ -31,12 +31,30 @@
  */
 @AutoValue
 public abstract class SecretContent {
-  public static SecretContent of(long id, long secretSeriesId, String encryptedContent, String hmac, ApiDate createdAt,
-                                 @Nullable String createdBy, ApiDate updatedAt, @Nullable String updatedBy,
-                                 ImmutableMap<String, String> metadata, long expiry) {
-    return new AutoValue_SecretContent(id, secretSeriesId, encryptedContent, hmac,
-        createdAt, nullToEmpty(createdBy), updatedAt,
-        nullToEmpty(updatedBy), metadata, expiry);
+  public static SecretContent of(
+      long id,
+      long secretSeriesId,
+      String encryptedContent,
+      String hmac,
+      ApiDate createdAt,
+      @Nullable String createdBy,
+      ApiDate updatedAt,
+      @Nullable String updatedBy,
+      ImmutableMap<String, String> metadata,
+      long expiry) {
+
+    return builder()
```

Comment:
For convenience, especially in constructing test data, this class now supports a builder. There are no additional fields.